### ISSUE TITLE
[Code] Create empty/dummy frontend project

### ISSUE THREAD

**Issue by mcarmenjc** [aislinggallagher/PluggableAuthGroup25#15]
- Frontend folder
- Scafolding of project
  - src (javascript)
  - image
  - css
  - entry point

**Comment by aislinggallagher** [comment]
https://medium.com/@adnanrahic/hello-world-app-with-node-js-and-express-c1eb7cfa8a30

**Comment by mcarmenjc** [comment]
> https://medium.com/@adnanrahic/hello-world-app-with-node-js-and-express-c1eb7cfa8a30

NodeJS is mainly used for backend development.

I'd suggest to start somewhere here:
- For ReactJS -> https://reactjs.org/docs/add-react-to-a-website.html
- For AngularJS -> https://angular.io/quick-start
- For Vue JS -> https://vuejs.org/guide/quick-start.html#creating-a-vue-application

**Comment by aislinggallagher** [comment]
I have experience using Node for front-end (we had a group project last semester built exactly like this). We can maybe discuss it on a call?

**Comment by mcarmenjc** [comment]
Yeah! I'm happy to go over the options on a call, just let me know when it's best for you
